__tests__/app.test.js

```diff
@@ -81,3 +82,52 @@ describe("GET /api/articles/:article_id", () => {
       })
   })
 })
+
+describe("GET /api/articles", () => {
+  const expectedObject = {
+    author: expect.any(String),
+    title: expect.any(String),
+    article_id: expect.any(Number),
+    topic: expect.any(String),
+    created_at: expect.any(String),
+    votes: expect.any(Number),
+    article_img_url: expect.any(String),
+    comment_count: expect.any(Number),
+  }
+  test("200: Responds with an array of article objects where the body", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeInstanceOf(Array)
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: